dev/provider_packages/prepare_provider_packages.py

```diff
@@ -951,14 +958,14 @@ def strip_leading_zeros(version: str) -> str:
     This converts 1974.04.03 to 1974.4.3 as the format with leading month and day zeros is not accepted
     by PIP versioning.
 
-    :param version: version number in calver format (potentially with leading 0s in date and month)
+    :param version: version number in CALVER format (potentially with leading 0s in date and month)
     :return: string with leading 0s after dot replaced.
     """
     return ".".join(str(int(i)) for i in version.split("."))
 
 
 def get_previous_release_info(
-    previous_release_version: str, past_releases: List[ReleaseInfo], current_release_version: str
+    previous_release_version: Optional[str], past_releases: List[ReleaseInfo], current_release_version: str
```

Comment:
same here

Reply:
Here we even cannot do it, because the following parameters have no defaults - so we can pass 'previous version as Null expliciltly, but it will never be default.